server/server.js

```diff
@@ -0,0 +1,44 @@
+require("dotenv").config();
+//utils imports
+const { RARITY } = require('./utils/constants');
+
+//services imports
+const { createItem } = require('./services/items-service')
+const { createOrnament } = require('./services/ornaments-service')
+const { createPlayer } = require('./services/players-service')
+const { createCreature } = require('./services/creatures-service')
+const { createCommunity } = require('./services/communities-service')
+const { initializeSettings } = require('./services/settings-service')
+
+//server dependancies
+
+const { token, databaseToken } = process.env; 
+const { connect, connection } = require('mongoose');
+
+console.log("INITIALIZING CHRISTMAS BOT");
+
+async function main() {
+    console.log("The elves are getting to work...")
+    await connect(databaseToken, {dbName: 'christmas_bot_v2'}).catch(console.error);
+    console.log("Christmas Bot Status: ONLINE");
+
+
+    //TESTING: set to true if you want to purge databases
+    let input = true
```

Comment:
What's this for?

Reply:
Oh oops, I see you were just moving the file and it was already there.
Still should do something about this, not sure what the intent of hard-coding this to true is

Reply:
@tangyyyyyyy to explain his reasoning

I think it's a dev-only flag so to purge the database when the code runs. I suppose the same can be accomplished by choosing to comment out/not comment out the code

Reply:
You can directly do
If (true) or if (false), doesn't need to be set into a variable.

But if a variable is gonna be used you should name it something other than input, maybe something like "purgeDatabaseOnStartup"

Going down train of thought, a few things to consider..

Could create a seedDatabase() function that drop purges the database on startup, but also creates fresh test data if needed that would be consistent everytime. Might not be the intent now, but at some point we'd probably want some existing test data without having to add it manually each time.